src/styles/blocks/card.css

```
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  --color-grey: #616070;
+  --color-blue: #00acdc;
+  --color-white: #fff;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-name {
+  margin: 40px 0 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__product-code {
+  margin: 4px 0 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-grey);
+}
+
+.card__product-evaluation {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__product-reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  align-self: flex-end;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-grey);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  text-decoration: none;
+  text-align: center;
+  padding: 11px 0;
+  margin-top: 16px;
+  text-transform: uppercase;
+  background-color: var(--color-blue);
+  border-radius: 5px;
+  border: 1px solid var(--color-blue);
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  color: var(--color-white);
+}
+
+.card__button:hover {
+  background-color: var(--color-white);
+  color: var(--color-blue);
+}
```

```suggestion
}

```
empty line on file ending